src/vs/workbench/contrib/preferences/browser/settingsSearchMenu.ts

```diff
@@ -107,7 +107,7 @@ export class SettingsSearchFilterDropdownMenuActionViewItem extends DropdownMenu
 				localize('extSettingsSearch', "Extension ID..."),
 				localize('extSettingsSearchTooltip', "Add extension ID filter"),
 				`@${EXTENSION_SETTING_TAG}`,
-				false
+				true
```

Comment:
This change makes it so the completions list activates after the user selects the option from the funnel button.